packages/graphiql/src/components/common/Resizer/ResizeHandler.tsx

```diff
@@ -0,0 +1,84 @@
+/** @jsx jsx */
+import { jsx } from 'theme-ui';
+import React, { useCallback, useEffect, useRef, useState } from 'react';
+import { ResizeHandlerData, ResizeHandlerProps } from './types';
+
+export const ResizeHandler: React.FC<ResizeHandlerProps> = props => {
+  const { dir, onStart, onEnd, onUpdate, className, style, children } = props;
+
+  const [isDragging, setIsDragging] = useState(false);
+  const listenersRef = useRef<ResizeHandlerData['listenersRef']>(null);
```

Comment:
would we want to use `forwardRef` here to allow users to pass `ref`? or do we only need it internally?